success.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
```

Comment:
Let be consistent here. Instead of having the style on the same page separate it to its own file as you did for the rest of the html docs.

Reply:
I didn't separate it cuz it's a bit style, so, I think no need 😅

Reply:
I didn't separate it cuz it's a bit style, so, I think no need 😅

Reply:
It's important to be consistent. Even in this case.

Reply:
Done!